### DISCUSSION TITLE
喜阳阳 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#440] [General]
# 喜阳阳 - Miao Yu | 于淼

https://yufree.cn/cn/2023/11/11/mian/

**Comment by KexingWestRoad** [comment]
现在才阳，难怪没看到你的博客更新了，看描述症状比我们去年冬天阳的时候轻很多了，还没有刀割喉的感觉，哈哈哈

**Reply by yufree** [reply]
有刀片嗓啊，只不过我一直在吃喉糖，这边喉糖有麻醉成分，吃上就缓解，不吃就复发，开始那几天早上起来就明显喉咙不适，也有咳痰。而且我症状轻不代表病程短，我吃的药都是缓解症状而非治疗，现在测抗原还是弱阳的，静息心率也没有恢复，只是我自己感受不明显。
